tests/foreman/cli/test_repositories.py

```diff
@@ -124,3 +124,41 @@ def test_positive_disable_rh_repo_with_basearch(module_target_sat, module_entitl
         }
     )
     assert 'Repository disabled' in disabled_repo[0]['message']
+
+
+def test_positive_reclaim_space(module_target_sat, module_entitlement_manifest_org):
+    """Hammer repository reclaim-space should not throw any improper exceptions
+
+    :id: 74b669d8-ee6b-4fc6-864f-91410d7ea3c2
+
+    :steps:
+        1. Enable and sync an On Demand repo
+
+        2. hammer repository reclaim-space --id REPOID --organization-id ORGID
+
+
+    :expectedresults: Command works as expected
```

Comment:
Should have a better expected result! Something like the space is reclaimed or more descriptive.

Reply:
Yes, it should state here that the space is really reclaimed and at the end, it should be asserted. The fact this passes without any error doesn't mean it works.